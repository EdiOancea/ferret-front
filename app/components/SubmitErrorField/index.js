import React from 'react';
import PropTypes from 'prop-types';

const SubmitErrorField = ({ errorMessage }) => <div> {errorMessage} </div>;

SubmitErrorField.propTypes = {
  errorMessage: PropTypes.string,
};

export default SubmitErrorField;
