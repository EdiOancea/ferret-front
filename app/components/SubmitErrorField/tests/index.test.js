import React from 'react';
import { render } from 'enzyme';
import SubmitErrorField from 'components/SubmitErrorField';

describe('<SubmitErrorField />', () => {
  it('should not regress', () => {
    const tree = render(<SubmitErrorField errorMessage="test error message" />);
    expect(tree).toMatchSnapshot();
  });
});
