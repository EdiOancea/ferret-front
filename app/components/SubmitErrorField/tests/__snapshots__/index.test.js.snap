// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitErrorField /> should not regress 1`] = `
<div>
   test error message 
</div>
`;
